src/components/header/header.js

```diff
@@ -21,19 +20,55 @@ const AnimatedContainer = posed.div({
 });
 
 const Header = () => {
+  const menuGroups = useStaticQuery(graphql`
+    query {
+      allMegaMenuGroup {
+        edges {
+          node {
+            name
+            url
+            menuColumns {
+              menuColumnGroups {
+                name
+                menuItems {
+                  name
+                  url
+                  description
+                  iconImg
+                }
+              }
+            }
+            sidebarItems {
+              name
+              items {
+                name
+                url
+                description
+                widgetType
+                icon
+              }
+            }
+            viewAll {
+              name
+              url
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  const menuItems = menuGroups.allMegaMenuGroup.edges.map((edge) => edge.node);
+
   return (
     <AnimatedContainer>
       <header>
-        <div className="flex justify-between items-center mx-2 md:mx-6 mt-4 mb-6">
-          <div className="flex items-center">
-            <a href={parentSiteUrl} className="unstyled cursor-pointer">
-              <CompanyLogo aria-label="logo" />
-            </a>
-            <h1 className="title ml-2">People</h1>
-          </div>
-          <div className="action-btn-container print-hidden">
-            <CountrySelect />
-          </div>
+        <div className="mx-2 md:mx-6 print-hidden no-underline">
+          <MegaMenuLayout
```

Comment:
Possible to change the behavior of when clicking "People" it goes back to `/people` instead of `/` (which goes to `ssw.com.au`)

Reply:
Blocked by https://github.com/SSWConsulting/SSW.MegaMenu/pull/61